### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by matt-oc**
Updated read me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,68 @@
...
+```sh
+npm run test
+```
+
+## Contributing
+The [Senecajs org](https://github.com/senecajs/) encourage open participation. If you feel you can help in any way, be it with
+documentation, examples, extra testing, or new features please get in touch.
+
 ## License
 Copyright Mircea Alexandru and other contributors 2016, Licensed under [MIT][].
```
Comment (mihaidma) [inline README.md]: Please double check the author, it could be Nicholas

File: README.md [hunk 2]
```diff
@@ -1,19 +1,68 @@
...
 
-Seneca auth uses passport.js.
+# seneca-linkedin-auth
 
+[![npm version][npm-badge]][npm-url]
+[![Build Status][travis-badge]][travis-url]
+[![Dependency Status][david-badge]][david-url]
+[![Gitter chat][gitter-badge]][gitter-url]
+
+Lead Maintainer: [Mircea Alexandru](https://github.com/mirceaalexandru)
```
Comment (mihaidma) [inline README.md]: no lead maintainer established yet for the package, please remove

File: README.md [hunk 3]
```diff
@@ -1,19 +1,68 @@
...
+documentation, examples, extra testing, or new features please get in touch.
+
 ## License
-Copyright Mircea Alexandru and other contributors 2016, Licensed under [MIT][].
+Copyright Nicolas Herment and other contributors 2016, Licensed under [MIT][].
 
 [MIT]: ./LICENSE
 [online]: http://htmlpreview.github.com/?https://github.com/senecajs/seneca-linkedin-auth/blob/master/docs/linkedin-auth.html
 [here]: ./docs/linkedin-auth.html
+[npm-badge]: https://badge.fury.io/js/seneca-local-auth.svg
```
Comment (ckiss) [inline README.md]: local-auth?!

File: README.md [hunk 4]
```diff
@@ -1,19 +1,67 @@
...
+[npm-url]: https://badge.fury.io/js/seneca-local-auth
+[david-badge]: https://david-dm.org/mirceaalexandru/seneca-local-auth.svg
```
Comment (ckiss) [inline README.md]: senecajs instead of mirceaalexandru

File: package.json [hunk 5]
```diff
@@ -9,7 +9,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git://linkedin.com/nherment/seneca-linkedin-auth.git"
+    "url": "git://linkedin.com/senecajs/seneca-linkedin-auth.git"
```
Comment (ckiss) [inline package.json]: linkedin.com should be github.com

**Comment by ckiss** [comment]
LGTM

**Pull request merged**